backend/migrations/versions/647cfc4c0735_add_test_case_outputs_table.py

```
@@ -1,40 +1,45 @@
 
 
 # revision identifiers, used by Alembic.
-revision = '647cfc4c0735'
-down_revision = '799a2343955e'
+revision = "647cfc4c0735"
+down_revision = "65e049e2083e"
 branch_labels = None
 depends_on = None
 
 
 def upgrade():
```

Ruff (0.8.2)

Missing return type annotation for public function `upgrade`

Add return type annotation: `None`

(ANN201)